packages/connect-voting-disputable/src/models/DisputableVoting.ts

```diff
@@ -0,0 +1,95 @@
+import { Address, SubscriptionHandler } from '@aragon/connect-types'
+
+import Vote from './Vote'
+import Voter from './Voter'
+import Setting from './Setting'
+import { IDisputableVotingConnector } from '../types'
+
+export default class DisputableVoting {
+  #address: Address
+  #connector: IDisputableVotingConnector
+
+  constructor(connector: IDisputableVotingConnector, address: Address) {
+    this.#connector = connector
+    this.#address = address
+  }
+
+  async disconnect() {
+    await this.#connector.disconnect()
+  }
+
+  async id(): Promise<string> {
+    const data = await this.#connector.disputableVoting(this.#address)
+    return data.id
+  }
+
+  onId(callback: Function): SubscriptionHandler {
+    return this.#connector.onDisputableVoting(this.#address, callback)
+  }
+
+  async dao(): Promise<string> {
+    const data = await this.#connector.disputableVoting(this.#address)
+    return data.dao
+  }
+
+  onDao(callback: Function): SubscriptionHandler {
```

Comment:
I wonder in which cases we may have a subscription to `onDao`? 🤔

Reply:
I was kind of inclined to do this due to Pierre's suggestion (see [here](https://github.com/aragon/connect/pull/165#discussion_r464546997)), however, maybe we can avoid those that may not be necessary

Reply:
Oh, cool, I think we can avoid some of them 👍